sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -98,15 +98,20 @@ export class ApplicationApi extends HttpBaseClient {
    * API Client for application detail.
    * @param applicationId for the application.
    * @param studentId for the student.
+   * @param isParentApplication if true, loads the parent application.
    * @returns application details.
    */
   async getApplicationDetails(
     applicationId: number,
     studentId?: number,
```

Comment:
In Assessment and other pages loaded from side nav which has a common component called application header. This application header calls the get application details API. We need it to call with parent application as true. 

![image](https://github.com/user-attachments/assets/422a1728-7d7d-452d-b767-9c694ce73b37)

Reply:
Here is the Vue service call starting point, and we can refactor from here. https://github.com/bcgov/SIMS/blob/f32a64efbf2f0b080b0a98aba223a0c6cf4da914/sources/packages/web/src/components/aest/students/ApplicationHeaderTitle.vue#L42